i3_resurrect/util.py

```diff
@@ -1,8 +1,22 @@
 import sys
 
-
 def eprint(*args, **kwargs):
     """
     Function for printing to stderr.
     """
     print(*args, file=sys.stderr, **kwargs)
+
+
+def filename_filter(filename):
+    """
+    Take a string and return a valid filename constructed from the string.
+"""
+    blacklist = '/\:*"<>|'
```

Comment:
Just noticed there's an unescaped backslash. Python lets you do this but it's not good practice, so you should put a double backslash `\\`